### ISSUE TITLE
question on the pretrained models 

### ISSUE THREAD

**Issue by jkhourybbn** [clovaai/voxceleb_trainer#145]
Hi all,

Can you please confirm that the two pretrained models provided here (`baseline_lite_ap.model` and `baseline_v2_ap.model`) were trained on development set of VoxCeleb2 and NOT on test set of VoxCeleb1, as specified in [1]? 

Many thanks

**Comment by msh9184** [comment]
By checking the reported performances in each paper [1,2], it seems that both models were trained through the dev set of VoxCeleb2.
Practically, when I trained both models using only vox2 dev set from scratch, similar performances were confirmed. :)
